client/web/compose/src/components/PageBlocks/Navigation/Configurator.vue

```diff
@@ -0,0 +1,291 @@
+<template>
+  <div>
+    <b-tab :title="$t('navigation.label')">
+      <div class="mb-3">
+        <h5 class="text-primary">
+          {{ $t("navigation.displayOptions") }}
+        </h5>
+
+        <b-row class="justify-content-between">
+          <b-col
+            sm="4"
+            class="mb-2 mb-sm-0"
+          >
+            <b-form-group
+              horizontal
+              :label="$t('navigation.appearance')"
+            >
+              <b-form-radio-group
+                v-model="options.display.appearance"
+                buttons
+                button-variant="outline-secondary"
+                size="sm"
+                name="appearance"
+                :options="appearanceOptions"
+              />
+            </b-form-group>
+          </b-col>
+
+          <b-col
+            sm="4"
+            class="mb-2 mb-sm-0"
+          >
+            <b-form-group
+              horizontal
+              :label="$t('navigation.alignment')"
+            >
+              <b-form-radio-group
+                v-model="options.display.alignment"
+                buttons
+                button-variant="outline-secondary"
+                size="sm"
+                name="alignment"
+                :options="alignmentOptions"
+              />
+            </b-form-group>
+          </b-col>
+
+          <b-col
+            sm="4"
+            class="mb-2 mb-sm-0"
+          >
+            <b-form-group
+              horizontal
+              :label="$t('navigation.fillJustify')"
+            >
+              <b-form-radio-group
+                v-model="options.display.fillJustify"
+                buttons
+                button-variant="outline-secondary"
+                size="sm"
+                :options="fillJustifyOptions"
+              />
+            </b-form-group>
+          </b-col>
+        </b-row>
+      </div>
+
+      <div class="mb-3 mt-2">
+        <div class="d-flex align-items-center mb-4">
+          <h5 class="text-primary mb-0">
+            {{ $t("navigation.listItems") }}
```

Comment:
in my opinion these aren't really "List items", they are more like navigation items, that would be the same for the class and properties in corteza-js. You could just call it Items, and since they are in navigation it makes sense they are navigation items